app/utils/computed-helpers.js

```diff
@@ -47,13 +47,13 @@ export const computedSegmentedLink = function(property) {
 export const computedDateTimeSplit = function(property, segmentFormat, endProperty) {
   return computed(property, {
     get() {
-      return moment(this.get(property)).format(getFormat(segmentFormat));
+      return moment(this.get(property)).tz(this.timezone).format(getFormat(segmentFormat));
```

Comment:
Will fail for everything except event

Reply:
@prateekj117 Dont send the timezone with time, as we are already collecting timezone in seperate field. I dont think sending timezone with date & time would be mandatory.

Reply:
It is mandatory. As I said, please read about moment and timezones and ISO 8601 format a bit more

Reply:
@iamareebjamal So adding a if conditional should work in that case here.

Reply:
Yes